src/extensions/exception-autocapture/stack-trace.ts

```diff
@@ -33,6 +33,7 @@ export type StackLineParserFn = (line: string) => StackFrame | undefined
 export type StackLineParser = [number, StackLineParserFn]
 
 export interface StackFrame {
+    lang: string
```

Comment:
i think this is same as `platform` (and useful to re-use imo when we extend because s£ntry SDKs _should_ populate this)

Reply:
Hmm good point. The docs aren't super clear
> This can override the platform for a single frame. Otherwise, the platform of the event is assumed. This can be used for multi-platform stack traces, such as in React Native.

My worry with platform is that it ends up populating something like "iPadOS", in which case we don't know whether it is a Swift or Objective-C frame 🤔

Reply:
Looking through the code it seems like `platform` is set to the language or something like `node`. In our instance the "platform" values are only ever `javascript` or `python`.

Will rename this and go with `platform` to maintain consistency but use serde to map from platform to lang when deserializing